packages/realm/src/Collection.ts

```diff
@@ -39,6 +41,20 @@ export abstract class Collection<T = unknown, ChangeCallbackType = unknown> impl
     throw new Error("Not yet implemented");
   }
 
+  toJSON(_?: string, cache = new Map()): Array<DefaultObject> {
```

Comment:
in here and in object `toJSON`, I used `DefaultObject` as expected return value. This to me feels like a way to make sure it is a "plain" object for sake of future JSON libraries but maybe there is a better way to go about it.